cloud/services/compute/network.go

```diff
@@ -81,10 +81,12 @@ func (s *Service) DeleteNetwork() error {
 	// Delete Network.
 	op, err := s.networks.Delete(s.scope.Project(), network.Name).Do()
 	if err != nil {
-		return errors.Wrapf(err, "failed to delete forwarding rules")
+		if !gcperrors.IsNotFound(err) {
```

Comment:
Similar to above, this conditional can be combined with the `err != nil` check above